src/cart/crt_internal_fns.h

```diff
@@ -32,8 +32,8 @@
 bool crt_initialized(void);
 
 /** crt_register.c */
-int crt_opc_map_create(unsigned int bits);
-void crt_opc_map_destroy(struct crt_opc_map *map);
+int crt_opc_map_create(void);
+void crt_opc_map_destroy();
```

Comment:
(style)  Bad function definition - void crt_opc_map_destroy() should probably be void crt_opc_map_destroy(void)